test/t8_data/t8_gtest_shmem.cxx

```diff
@@ -190,29 +100,25 @@ t8_test_sc_shmem_alloc (sc_MPI_Comm comm)
   /* Seed random number generator. */
   srand (0);
 
+  /* Checking shared memory type */
   for (int shmem_type_int = SC_SHMEM_BASIC;
        shmem_type_int < SC_SHMEM_NUM_TYPES; ++shmem_type_int) {
     const sc_shmem_type_t shmem_type = (sc_shmem_type_t) shmem_type_int;
     int                 intrasize, intrarank;
-    t8_productionf ("Checking shared memory type %s.\n",
```

Comment:
Please convert to `t8_debugf`.

Reply:
If you have a different opinion about this you can answer to my comment and explain. :)